docs/sql-ref-ansi-compliance.md

```diff
@@ -151,7 +151,7 @@ CREATE TABLE t (v INT);
 
 -- `spark.sql.storeAssignmentPolicy=ANSI`
 INSERT INTO t VALUES ('1');
-org.apache.spark.sql.AnalysisException: Cannot write incompatible data to table '`default`.`t`':
+org.apache.spark.sql.AnalysisException: Cannot write incompatible data for table '`default`.`t`':
 - Cannot safely cast 'v': string to int;
```

Comment:
The message should look differently after your changes. Could you re-run the example, and change the output.